inc/elementor/kit/Manager.php

```diff
@@ -116,7 +117,16 @@ public function is_using_custom_kit() {
 		}
 
 		// Return early if Global kit and current kit is same.
-		if ( (int) Options::get_instance()->get( 'global_kit' ) === (int) $kit_id ) {
+		$global_kit = Options::get_instance()->get( 'global_kit' );
+		if ( ! $global_kit ) {
+
+			$el_manager = new EL_Manager();
+			$global_kit = $el_manager->get_active_id();
```

Comment:
Could rewrite as following?

```suggestion
			$global_kit = ( new EL_Manager() )->get_active_id();
```